features/speech-grammar.yml

```diff
@@ -0,0 +1,14 @@
+name: Speech grammar
+description: The `SpeechGrammar` interface of the Web Speech API represents a set of words or patterns of words for the `SpeechRecognition` service to recognize.
```

Comment:
This text (and the MDN docs) strongly imply that you wouldn't use this API alone—you'd only ever use it with `SpeechRecognition`. I'd combine this with SpeechRecognition.